editor/editor_data.cpp

```diff
@@ -569,6 +569,7 @@ bool EditorData::_find_updated_instances(Node *p_root, Node *p_node, HashSet<Str
 
 			checked_paths.insert(path);
 		}
+		ss = ss->get_base_scene_state();
```

Comment:
Due to #57606, `get_base_scene_state()` gets the latest version of `base_scene_state()`.